grunt/tasks/options/jscs.coffee

```diff
@@ -0,0 +1,12 @@
+loadLocals = require('../../tools/loadLocals')
```

Comment:
Another improvement could be to use [load-grunt-config](https://github.com/firstandthird/load-grunt-config) to use a common library of grunt instead of our own grunt process which look alike.